calibimg.rst

```diff
@@ -51,28 +53,28 @@ Flags and utility variables
 
 -``doDeblend`` (`bool`) - defaults to `True` - Run deblender input exposure?
 	
------------
+
 
 Subtasks
 --------
 
-- 	``refObjLoader`` - target=LoadAstrometryNetObjectsTask -   reference object loader
+- 	``refObjLoader`` - default target=LoadAstrometryNetObjectsTask -   reference object loader
```

Comment:
I used the words 'default target' for all subtasks, to make it clear that it's the default, and not just a variable, but a Task its pointing at, thought this would make it clearer..?